example/test/example_test.dart

```diff
@@ -0,0 +1,10 @@
+// needed to make CI pass
+
+import 'package:flutter_test/flutter_test.dart';
+
+void main(){
+
+  test('Some test to make CI pass', (){
+    expect(1,1);
+  });
+}
```

Comment:
Why is this useful? I wouldn't add such stub files.

Reply:
because if there is no test in the example app folder the tooling on the CI fails so I need to stub it :(